### PR TITLE
Fix DeployStatus showing error when state is processing

### PR DESCRIPTION
**Pull request by martijngastkemper**

### CODE REVIEW

File: resources/js/components/DeployStatus.vue [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
       }).format(new Date(date));
     },
     isPublishing(state) {
-      return state === "building" || state === "new" || state === "enqueued";
+      return state !== "ready";
```
Comment (harmenjanssen) [inline resources/js/components/DeployStatus.vue]: Just checking, maar kan het voorkomen dat de state in error is, maar door deze aanpassing als _"is publishing"_ gezien wordt?

**Review by harmenjanssen — APPROVED** [review]
LGTM

**Comment by martijngastkemper** [comment]
You're right the error state wasn't handled properly. I've moved the state checks to separate functions now every state is covered. And I think it is easier to understand.

**Review by harmenjanssen — APPROVED** [review]
LGTM!

**Pull request merged**